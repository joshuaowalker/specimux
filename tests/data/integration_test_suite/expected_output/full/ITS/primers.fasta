>ITS1F position=forward pool=ITS
CTTGGTCATTTAGAGGAAGTAA
>ITS4 position=reverse pool=ITS,ITS2
TCCTCCGCTTATTGATATGC
