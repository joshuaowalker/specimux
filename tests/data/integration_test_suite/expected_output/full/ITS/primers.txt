>ITS1F
CTTGGTCATTTAGAGGAAGTAA
>ITS4
TCCTCCGCTTATTGATATGC
